argostime/crawler/crawl_utils.py

```diff
@@ -69,6 +76,52 @@ def __str__(self) -> str:
 
         return string
 
+    def check(self) -> None:
+        """Check if CrawlResult contains the mandatory data."""
```

Comment:
Misschien is het handig om hier ook aan toe te voegen _wanneer_ je deze method gebruikt

Ook: wat is mandatory data? Misschien handig om dat ook duidelijk te maken, dan maak je in een keer de code ook duidelijker (en hopelijk helpt het met het consistenter maken van de codebase)